utils/manifest-utils/Cargo.toml

```diff
@@ -9,6 +9,16 @@ license = "LGPL-3.0"
 description = "Yagna manifest utils"
 keywords = ["golem", "yagna"]
 
+[[bin]]
+name = "schema"
+path = "src/schema_bin.rs"
```

Comment:
`schema_bin` got used to generate schema added to [`yagna-doc`](https://github.com/golemfactory/yagna-docs/pull/130/files#diff-b7deae5160e10673e2df33606bd2c8dd26eaaf630f07541f658fc5b560f18931)
It should be decided if we want to create a Gihub Action generating and publishing schema.